### DISCUSSION TITLE
[Feature Request]: 部分系统服务执行间隔不能配置且运行间隔时间较短 希望能够支持自定义配置

### DISCUSSION THREAD

**Discussion by sfwilp** [jxxghp/MoviePilot#3481] [讨论]
### 当前程序版本

v1.9.0

### 运行环境

Docker

### 功能改进类型

其他

### 功能改进

[系统] 订阅刷新 等待 14秒  
[系统] 新增订阅搜索 等待 1分钟  
[系统] 下载文件整理 等待 1分钟  
[系统] 公共定时服务 等待 6分钟  
[系统] 用户认证检查 等待 6分钟  
[系统] 壁纸缓存 等待 26分钟 

以上6个服务执行间隔不能自定义   且运行间隔时间短   希望能够支持自定义运行时间间隔

**Comment by cddjr** [comment]
v2有插件可以修改了
